components/Calendar/index.tsx

```diff
@@ -0,0 +1,44 @@
+'use client'
+import FullCalendar from '@fullcalendar/react'
+import dayGridPlugin from '@fullcalendar/daygrid' // a plugin!
+import googleCalendarPlugin from '@fullcalendar/google-calendar'
+import SectionHeader from '../Common/SectionHeader'
+
+const CalendarComponent = ({calendarID,calendarApiKey} : {calendarID:string | undefined,calendarApiKey:string | undefined}) => {
```

Comment:
Maybe add a tab to change the view to week view. Day and year probably aren't as useful, so I wouldn't worry too much about these.